codalab/lib/bundle_cli.py

```diff
@@ -1195,6 +1195,13 @@ def do_workers_command(self, args):
                 action='store_true',
                 default=False,
             ),
+            Commands.Argument(
+                '-a',
+                '--use_azure_blob_beta',
```

Comment:
I think our convention is to use `-` rather than `_` for all command-line args.

Reply:
Done